_site/index.html

```diff
@@ -4,5 +4,17 @@
     <link rel="stylesheet" href="/stylesheets/styles.css">
   </head>
   <body>
+    <section class="hero">
+  <img src="images/logo.svg" alt="Ember and Rails"/>
```

Comment:
Everything in the `_site` dir, is compiled by `jekyll build`. I would rather avoid dealing with the indentation in any of those files.